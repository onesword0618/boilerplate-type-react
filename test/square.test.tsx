import React from 'react';
import renderer from 'react-test-renderer';
import Square from '../src/square';
describe('Squareの検証', () => {
  test('表示がスナップショットと一致すること', () => {
    const sut = renderer
      .create(
        <Square
          value="X"
          onClick={(): void => {
            'O';
          }}
        />
      )
      .toJSON();
    expect(sut).toMatchSnapshot();
  });
});
