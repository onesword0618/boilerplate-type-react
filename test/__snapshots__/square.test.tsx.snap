// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Squareの検証 表示がスナップショットと一致すること 1`] = `
<button
  className="square"
  onClick={[Function]}
>
  X
</button>
`;
